src/formatDate.js

```diff
@@ -8,7 +8,70 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  if (date !== undefined) {
+    const valuesOfDate = date.split(fromFormat[3]);
+
+    let DD = 0;
+    let MM = 0;
+    let year = 0;
```

Comment:
Initializing `DD`, `MM`, and `year` with `0` is misleading because they are meant to hold string values. It would be better to initialize them as empty strings.